src/App.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { BrowserRouter, Link, Route, Routes } from 'react-router-dom';
 import './App.css';
+import JourneySelector from './components/JourneySelector';
 import Station from './components/Station';
 import Stations from './components/Stations';
 
 const App = () => (
     <BrowserRouter>
         <div className = "App">
+            < JourneySelector />
```

Comment:
It's a bit confusing but there's no need for the space at the start, we can just say `<JourneySelector />` like `<div>` the 
`  />` is just to represent it is self closing and has no separate closing tag